lib/install/management/appliance.go

```diff
@@ -298,7 +298,8 @@ func (d *Dispatcher) createApplianceSpec(conf *config.VirtualContainerHostConfig
 	spec := &spec.VirtualMachineConfigSpec{
 		VirtualMachineConfigSpec: &types.VirtualMachineConfigSpec{
 			Name:     conf.Name,
-			GuestId:  "other3xLinux64Guest",
+			GuestId:  string(types.VirtualMachineGuestOsIdentifierOtherGuest64),
+			AlternateGuestName: "vSphere Integrated Containers - VCH 0.9.0",
```

Comment:
^^^ I think we should not hard code the version number.
In addition, besides of the release version, should we add more details, like build number? Which can make the telemetry more accurate.